tests/product/tests/data/k8s_object/eks_k8s_object_test_cases.py

```diff
@@ -160,3 +161,63 @@
     '4.2.9 PSP spec.securityContext.capabilities.drop==["ALL"] eval passed': cis_eks_4_2_9_pass,
     '4.2.9 PSP spec.securityContext.capabilities.add==["NET_ADMIN", "SYS_TIME"] eval failed': cis_eks_4_2_9_fail,
 }
+
+cis_eks_all = {
```

Comment:
would creating each dict separately (`test-eks-config-1`, `test-eks-config-2`) will remove the need of unpacking and zipping? it's a bit complex IMO.

Reply:
something like:

```python
test_eks_config_1 = {
    **cis_eks_4_2_7,
    **cis_eks_4_2_8,
    **cis_eks_4_2_9,
}

test_eks_config_2 = {
    **cis_eks_4_2_1,
    **cis_eks_4_2_2,
    **cis_eks_4_2_3,
    **cis_eks_4_2_4,
    **cis_eks_4_2_5,
    **cis_eks_4_2_6,
}

cis_eks_all = {
    "test-eks-config-1": test_eks_config_1,
    "test-eks-config-2": test_eks_config_2,
}
```

Reply:
As I described before, zip is used when we need to skip the test case.